elyra/pipeline/pipeline.py

```diff
@@ -175,6 +181,7 @@ def __eq__(self, other: object) -> bool:
             return self.id == other.id and \
                 self.type == other.type and \
                 self.classifier == other.classifier and \
+                self.name == other.name and \
```

Comment:
This method needs a `return False` at the end in cases where the two classes differ; otherwise, the result is `None` instead of `False`.

Reply:
added